manager/integration/tests/test_provisioner.py

```diff
@@ -63,6 +63,23 @@ def test_provisioner_mount(client, core_api, storage_class, pvc, pod):  # NOQA
     assert volumes.data[0].state == "attached"
 
 
+@pytest.mark.skip(reason="TODO")
+def test_provisioner_generic_ephemeral():
+    """
+    Test that a Longhorn generic ephemeral volume can be created, mounted,
+    unmounted, and deleted properly on the Kubernetes cluster.
+
+    1. Create a StorageClass and Pod with a generic ephemeral volume spec:
+       https://kubernetes.io/docs/concepts/storage/ephemeral-volumes
+    2. Verify:
+       - The Pod is running.
+       - The volume parameters match the StorageClass parameters.
+       - The volume.status.kubernetesStatus.workloadStatus reflects the
+         running Pod.
+    """
```

Comment:
Should we add a test step attempting to write data in ephemeral volume?

Reply:
It definitely can't hurt, so I added it in an amend. For context, I wrote this skeleton to mirror `test_provisioner_mount` (directly above in the file), which does not. Importantly, we cannot follow the pattern of some other tests and read the data back from a different pod, because the life cycle of a generic ephemeral volume mirrors the life cycle of the pod it is created in.